.eslintrc

```diff
@@ -1,10 +1,11 @@
 {
-  "extends": "rapid7/base",
+  "extends": "rapid7/node",
```

Comment:
Will do. I'll open one for this and for the `invalidate()` method and then redo this one.